web/src/assets/index.ts

```diff
@@ -1,76 +1,98 @@
 /* eslint-disable import/no-relative-packages */
+import EventThumbnailPlaceholder1 from '../../../assets/event-thumbnail-placeholder-1.jpg'
+import EventThumbnailPlaceholder2 from '../../../assets/event-thumbnail-placeholder-2.jpg'
+import EventThumbnailPlaceholder3 from '../../../assets/event-thumbnail-placeholder-3.jpg'
 import AccessibleIcon from '../../../assets/icons/accessible.svg'
-import ArrowBackIcon from '../../../assets/icons/arrow_back.svg'
+import ArrowBackIcon from '../../../assets/icons/arrow-back.svg'
+import ArrowBackspaceWhiteIcon from '../../../assets/icons/arrow-backspace-white.svg'
+import ArrowBackspaceIcon from '../../../assets/icons/arrow-backspace.svg'
 import BathroomIcon from '../../../assets/icons/bathroom.svg'
 import BedIcon from '../../../assets/icons/bed.svg'
 import CalendarIcon from '../../../assets/icons/calendar.svg'
-import CloseSharingIcon from '../../../assets/icons/close-sharing.svg'
+import CategoriesIcon from '../../../assets/icons/categories.svg'
+import CloseGreyIcon from '../../../assets/icons/close-grey.svg'
 import CloseIcon from '../../../assets/icons/close.svg'
 import CopyIcon from '../../../assets/icons/copy.svg'
 import DoneIcon from '../../../assets/icons/done.svg'
+import EmailGreyIcon from '../../../assets/icons/email-grey.svg'
 import EmailIcon from '../../../assets/icons/email.svg'
 import EuroIcon from '../../../assets/icons/euro.svg'
 import EventsIcon from '../../../assets/icons/events.svg'
+import ExternalLinkIcon from '../../../assets/icons/external-link.svg'
 import FacebookIcon from '../../../assets/icons/facebook.svg'
 import FeedbackIcon from '../../../assets/icons/feedback.svg'
+import HappySmileyIcon from '../../../assets/icons/happy-smiley.svg'
 import HouseIcon from '../../../assets/icons/house.svg'
 import KeyIcon from '../../../assets/icons/key.svg'
 import LanguageIcon from '../../../assets/icons/language.svg'
 import LgbtqiIcon from '../../../assets/icons/lgbtqi.svg'
 import LocationIcon from '../../../assets/icons/location.svg'
-import MailSocialIcon from '../../../assets/icons/mail-social.svg'
-import NegativeFeedbackIcon from '../../../assets/icons/negative-feedback.svg'
+import MenuIcon from '../../../assets/icons/menu.svg'
 import NewsIcon from '../../../assets/icons/news.svg'
 import NoteIcon from '../../../assets/icons/note.svg'
 import OffersIcon from '../../../assets/icons/offers.svg'
 import PdfIcon from '../../../assets/icons/pdf.svg'
 import PetIcon from '../../../assets/icons/pet.svg'
 import PhoneIcon from '../../../assets/icons/phone.svg'
 import POIsIcon from '../../../assets/icons/pois.svg'
-import PositiveFeedbackIcon from '../../../assets/icons/positive-feedback.svg'
+import SadSmileyIcon from '../../../assets/icons/sad-smiley.svg'
```

Comment:
Just a thought: We are somehow inconsistent with the icon namings, sometimes we name them depending on their content (what can you see, e.g. SadSmileIcon) and sometimes on their usage (e.g. SearchIcon).

Reply:
Do you have any preference? Someone else?

Reply:
No preference, but consistency would be nice, currently we mostly use naming on their usage (semantic). Pros and cons are for semantic naming are:
pros: - easy to change a icon, if the design is updated (no renaming needed)
cons: - use icon for multiple use cases will break the naming convention, e.g. when we use the sad smile for giving negative feedback and also for no search results
As the pro will not happen that often and a renaming is not really hard as we only import the file in the index file, i have a tendency to use naming on the content (syntactical), this is also how we do this in Lunes.

But as most icons are named semantically, maybe it is to much overhead for this ticket? If you do not want to do it here open a ticket or let me know and i will open one.

Reply:
Lets do it in another PR. I am also in favor of syntactically.